doc/environment.yml

```diff
@@ -0,0 +1,6 @@
+name: nest-doc
+dependencies:
+  - python=3.6
```

Comment:
Why do you require Py 3.6 specifically?

Reply:
This is what we had in the previous incarnation of the `readthedocs.yml` file because we use 3.6 features in the documentation extraction pipeline. As we could not express a _minimal_ requirement in that file, I put in the fixed version. I'm open to replacing the `=` by `>=` now and just go with whatever the local conda installation provides.